app/controllers/institutions_controller.rb

```diff
@@ -36,6 +36,35 @@ def update
     end
   end
 
+  def merge
+    @title = I18n.t('institutions.merge.title', name: @institution.name)
+    @institutions = apply_scopes(policy_scope(Institution))
+                    .where.not(id: @institution.id)
+                    .order(generated_name: :desc, name: :asc)
+                    .includes(:courses, :users, :providers)
+                    .paginate(per_page: 15, page: parse_pagination_param(params[:page]))
+  end
+
+  def merge_changes
+    @other = Institution.find(params[:other_institution_id])
+    authorize @other
+  end
+
+  def do_merge
+    @other = Institution.find(params[:other_institution_id])
+    authorize @other
+    if @institution.merge_into(@other)
+      redirect_to institution_url(@other), notice: I18n.t('views.institutions.merge.done')
```

Comment:
```suggestion
      redirect_to institution_url(@other), notice: I18n.t('institutions.merge.done')
```
Otherwise this gives translation errors for me (on mestra & locally).